virttest/nfs.py

```diff
@@ -225,7 +225,7 @@ def setup(self):
         if not os.path.isdir(self.mount_dir):
             os.makedirs(self.mount_dir)
             self.rm_mount_dir = True
-        self.mount()
+        utils_misc.wait_for(self.mount(), 5, text="Waiting for nfs mount")
```

Comment:
for retry, you could use 'soft' mode of 'mount' option like : 'mount -o soft,timeo=20,retrans=3'.